### PR TITLE
Add ResNet18 accuracy checker config

### PR DESCRIPTION
**Pull request by vshampor**

### CODE REVIEW

File: tools/accuracy_checker/configs/resnet-18.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+models:
+  - name: resnet-18
```
Comment (vladimir-dudnik) [inline tools/accuracy_checker/configs/resnet-18.yml]: if 'v1' is significant suffix, I'd prefer to have that consistent in model name and model folder. Do you have any objections agains name like resnet-18-v1?

File: tools/accuracy_checker/configs/resnet-18.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+    launchers:
+      - framework: onnx_runtime
+        model: public/resnet-18/caffe2/resnet-v1-18.onnx
```
[the remark this answers is not in the document]
Comment (vshampor) [reply tools/accuracy_checker/configs/resnet-18.yml]: What if we have the same resnet-18 available for multiple frameworks? For caffe and caffe2, for instance.

[the remark this answers is not in the document]
Comment (eaidova) [reply tools/accuracy_checker/configs/resnet-18.yml]: then used postfix in model name

[the remark this answers is not in the document]
Comment (eaidova) [reply tools/accuracy_checker/configs/resnet-18.yml]: As example, you can look on resnet-50, in omz we have:
[resnet-50](https://github.com/opencv/open_model_zoo/tree/develop/models/public/resnet-50) - for caffe model
[resnet-50-pytorch](https://github.com/opencv/open_model_zoo/tree/develop/models/public/resnet-50-pytorch) - for pytorch model
[resnet-50-caffe2](https://github.com/opencv/open_model_zoo/tree/develop/models/public/resnet-50-caffe2) - for caffe2 version

File: models/public/resnet-18-pytorch/model.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  matching with those in the ImageNet database.
+task_type: classification
+files:
+  - name: resnet18-5c106cde.pth
+    sha256: 5c106cde386e87d4033832f2996f5493238eda96ccf559d1d62760c4de0613f8
+    size: 46827520
+    source: https://download.pytorch.org/models/resnet18-5c106cde.pth
+framework: pytorch
+conversion_to_onnx_args:
+  - --model-name=resnet18
```
Comment (eaidova) [inline models/public/resnet-18-pytorch/model.yml]: name used MO for xml/bin files naming, here should be `resnet18-pytorch`, another way you got  `public/resnet-18-pytorch/resnet-18.xml`

File: models/public/resnet-18-pytorch/model.yml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  matching with those in the ImageNet database.
+task_type: classification
+files:
+  - name: resnet18-5c106cde.pth
+    sha256: 5c106cde386e87d4033832f2996f5493238eda96ccf559d1d62760c4de0613f8
+    size: 46827520
+    source: https://download.pytorch.org/models/resnet18-5c106cde.pth
+framework: pytorch
+conversion_to_onnx_args:
+  - --model-name=resnet-18
```
Comment (eaidova) [inline models/public/resnet-18-pytorch/model.yml]: according to torchvision code https://github.com/pytorch/vision/blob/master/torchvision/models/resnet.py#L6
```suggestion
  - --model-name=resnet18
```

**Comment by vladimir-dudnik** [comment]
ok, changes looks good. Do not merge this until R4 is done. It is content of the next release.

**Comment by IRDonch** [comment]
Jenkins please retry a build

**Pull request merged**